.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
         - if [[ ! -f ~/venv/bin/activate ]]; then virtualenv -p python ~/venv; fi
         - source ~/venv/bin/activate
         - pip install --disable-pip-version-check --upgrade pip==8.1.1
+      exclude:
+        tests/jenkins/
```

Comment:
To exclude this you'll need to add the path to the `setup.cfg` file for [pycodestyle](https://github.com/mozilla/treeherder/blob/bb81039e58b8363f555cf89095b7bbbd703023db/setup.cfg#L2), [flake8](https://github.com/mozilla/treeherder/blob/bb81039e58b8363f555cf89095b7bbbd703023db/setup.cfg#L15), and [isort](https://github.com/mozilla/treeherder/blob/bb81039e58b8363f555cf89095b7bbbd703023db/setup.cfg#L23).